cranelift/codegen/src/isa/riscv64/inst/mod.rs

```diff
@@ -624,12 +624,21 @@ fn riscv64_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut Operan
             // gen_prologue is called at emit stage.
             // no need let reg alloc know.
         }
-        &Inst::VecAluRRR { vd, vs1, vs2, .. } => {
+        &Inst::VecAluRRR {
+            op, vd, vs1, vs2, ..
+        } => {
+            debug_assert_eq!(vd.to_reg().class(), RegClass::Float); // TODO: should be Vector
+            debug_assert_eq!(vs2.class(), RegClass::Float); // TODO: should be Vector
```

Comment:
What needs to change for these to be converted to `Vector`?

Reply:
Nothing really, just waiting on https://github.com/bytecodealliance/wasmtime/pull/6366 to be merged.